### DISCUSSION TITLE
Is it possible use cronjob to Send stored emails created by PhpMailer Without calling HTTP ?

### DISCUSSION THREAD

**Discussion by Joaquim3** [PHPMailer/PHPMailer#2549] [General]
Hello,

Is it possible use cronjob to Send stored emails created by PhpMailer Without calling HTTP ?

/************** my scenario  ***************************************/
1. I edit a webbased Newsletter (up to 5000 emails)
2. I click SEND
3. The Newsletter emails contents are all stored in a table in the database but are NOT sent. 
/*****************************************************/

I created a cronjob that calls an HTTP file "sendmails.php" every 1 second to SEND the stored emails
*/1 * * * * /usr/bin/curl -o temp.txt http://localhost/sendmails.php
/*****************************************************/
 
I would like to know if there is Another way to call PhpMailer than using HTTP in my cronjob command line 
Or am I obliged to call http://localhost/sendmails.php ?

Kind Regards,

**Comment by Synchro** [comment]
Yes. PHPMailer has nothing special to do with HTTP and there is no requirement to use it in a web context at all. Tell your cron job to run the script directly, something like:

    */1 * * * * /usr/bin/php /path/to/script/sendmails.php

**Reply by Joaquim3** [reply]
Thanks Synchro :-)
